### PR TITLE
supervisor/cli: rebuild external docs

### PR DESCRIPTION
**Pull request by xginn8**
Change-type: patch
Signed-off-by: Matthew McGinn <matthew@balena.io>

### CODE REVIEW

File: pages/reference/cli.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 The balena CLI (Command-Line Interface) allows you to interact with the balenaCloud and the
 [balena API](https://www.balena.io/docs/reference/api/overview/) through a terminal window
-on Linux, MacOS or Windows. You can also write shell scripts around it, or import its Node.js
+on Linux, macOS or Windows. You can also write shell scripts around it, or import its Node.js
```
Comment (pdcastro) [inline pages/reference/cli.md]: This is odd... The website at the moment already shows the "changes" being "introduced" in this PR, at least for the CLI docs: https://www.balena.io/docs/reference/cli/

Comment (pdcastro) [reply pages/reference/cli.md]: For the supervisor as well...

Comment (xginn8) [reply pages/reference/cli.md]: Hm, so it is! I don't know why that would be, I was under the impression this PR was required (obviously). I guess I should merge this PR to stay consistent, though presumably it does not matter too much..

**Comment by xginn8** [comment]
@balena-ci retest

**Pull request closed without merging**